src/Compute/Compute/ChangeLog.md

```diff
@@ -20,6 +20,7 @@
 
 -->
 ## Upcoming Release
+* Added Disk Delete Optional parameter to the `Set-AzVmssStorageProfile` (OS Disk) and `Add-AzVmssDataDisk` (Data Disk)
```

Comment:
let's write the actual parameter like the example below

Reply:
line 26/27 in this file

Reply:
Good catch will do